docs/install.rst

```diff
@@ -2,17 +2,21 @@
 Installation
 ************
 
+.. _astropy-main-req:
+
 Requirements
 ============
 
 Astropy has the following strict requirements:
 
-- `Python <https://www.python.org/>`_ 3.5 or 3.6
+- `Python <https://www.python.org/>`_ 3.5 or later
 
-- `Numpy`_ |minimum_numpy_version| or later
+- `Numpy`_ |minimum_numpy_version| or later (but not 1.15.3)
```

Comment:
I wonder if this is too specific - after all *most* of astropy does work fine with this version, and there may be other bug fix releases of numpy where some tests fail.

Reply:
It is not even the tests. RTD could not build without the `!=` pinning. I am open to suggestions though. https://readthedocs.org/projects/astropy/builds/8024893/

Reply:
Currently this is useful to have as people started to open issues they run into using astropy with 1.15.3. 

When the new numpy release is out, I would do agree with @astrofrog here that this is too specific (we had issues with other minor bugfix versions before, and never listed them once newer releases became available.)